src/export/text.c

```diff
@@ -94,7 +94,7 @@ text_disp_header(void)
 
 	/* use color option if triggered */
 	if (cflag)
-		(void)printf("\033[;%dm", cnf.chead);
+		(void)printf("\033[%d;%dm", cnf.lightcol, cnf.chead);
```

Comment:
What you call "light color" is actually just using bold font. This is confusing.